### ISSUE TITLE
Simpilfied mode

### ISSUE THREAD

**Issue by Lyapsus** [alkyaly/Somnia-Fabric#3]
How about to make it possible to turn off all fatique mechanics for only "sped up world" effect to remain?

**Comment by ejektaflex** [comment]
Yes, that's what I'd like as well. I really don't care for the fatigue aspect - Minecraft already has phantoms to make you sleep :)
